src/credentials/tests/BUILD.gn

```diff
@@ -46,14 +46,18 @@ chip_test_suite("tests") {
   test_sources = [
     "TestCertificationDeclaration.cpp",
     "TestChipCert.cpp",
-    "TestCommissionerDUTVectors.cpp",
     "TestDeviceAttestationConstruction.cpp",
     "TestDeviceAttestationCredentials.cpp",
     "TestFabricTable.cpp",
     "TestGroupDataProvider.cpp",
     "TestPersistentStorageOpCertStore.cpp",
   ]
 
+  # OIS platform does not support <dirent.h>
```

Comment:
Lets not name platforms, instead say `DUTVectors test requires <dirent.h> which is not supported on all platforms`